src/index.js

```diff
@@ -6,10 +6,14 @@ import Markdown from 'remarkable';
 class Remarkable extends React.Component {
 
   render() {
-    var Container = this.props.container;
+    var { container: Container, ...props } = this.props;
+
+    ['children', 'options', 'source'].forEach(prop => {
```

Comment:
Should `container` be added here as well?

Reply:
No need, it's not part of `props` due to the destructuring above.

Reply:
Oops, I'm blind. Carry on 👍 
Would love to see this merged soon - was just about to make a PR as well until I saw yours :)